analyses/cms-open-data-ttbar/ntuples_nanoaod.json

```diff
@@ -0,0 +1,3256 @@
+{
```

Comment:
Let's call this file "nanoaod_inputs.json" or something like that, the "ntuples" name might be slightly misleading.

Reply:
Done!